plugins.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Plugins
+headline: Pico Plugins
+description: |
+  Want to extend the functionality of Pico?  Below is a collection of our community-developed plugins.<br>
+  Would you like to create your own plugin? Learn how in [our documentation](/docs/#plugins)!
+nav: 5
+
+portfolio:
+  layout: card
+  categories:
+    admin: Administration
+    developer: Developer-Focused
+    theming: Theming Extensions
+  defaultThumbnail: plugins/images/thumbnails/plugin.png
+  defaultImage: themes/images/fullsize/default_theme.png
+
+---
+
+{% include portfolio.html portfolio=site.plugins %}
+
+---
+
+## More Plugins & How to Contribute Your Own
+
+The plugins shown here are only a smaller selection of those available.  You can find more themes on [our Wiki][].
+
+You can even [contribute][] your own plugin to the community and we'll feature it here.  While you can freely add your plugin to [our Wiki][], if you'd also like to be included on this page, you can follow the simple instructions in our [Submission Guidelines][].
```

Comment:
> I do think it would be a little hard to read though. Especially with white text on black. (And changing it to somehow have a white background inside the content area sounds odd).

Yeah, that's probably true.

> I'm thinking maybe make it a regular item, with a special fullsize image and description pointing to the regular Submission Guidelines page.
>
> For the fullsize image, maybe something like a collage of themes and code or something split diagonally between a theme and its code [T/C].

Sounds good! :+1: Can you create some sort of a mock-up or draft? So that we can see how it "feels", not that you're putting much work into it and it doesn't feel right in the end.

> Finally, I don't like the idea of removing the existing "Contributing" section from the page. Not sure if you meant that this "boring" text should be removed or not, but my vote is no.

No, that's not what I was trying to say, I commented just on some random line :laughing: